process/peer/process.go

```diff
@@ -270,6 +270,7 @@ func (vs *validatorStatistics) UpdatePeerState(header data.HeaderHandler) ([]byt
 
 	previousHeader, err := process.GetMetaHeader(header.GetPrevHash(), vs.dataPool.Headers(), vs.marshalizer, vs.storageService)
 	if err != nil {
+		log.Debug("process.GetMetaHeader", "error", err.Error(), "hash", header.GetPrevHash(), "round", header.GetRound(), "nonce", header.GetNonce())
```

Comment:
UpdatePeerState.GetMetaHeader

Reply:
changed message to "UpdatePeerState after process.GetMetaHeader"